src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,61 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import React from 'react';
+import { getSearchWith } from '../utils/searchHelper';
+
+const centuriesValue: string[] = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const resetParams = {
+    query: null,
+    sex: null,
+    centuries: null,
```

Comment:
The `resetParams` object uses `null` values, which might not be compatible with `setSearchParams`. Consider using empty strings or arrays to reset the parameters correctly.